pulp-glue/pulp_glue/rpm/context.py

```diff
@@ -259,6 +275,16 @@ def preprocess_entity(self, body: EntityDefinition, partial: bool = False) -> En
         body = super().preprocess_entity(body, partial=partial)
         if "autopublish" in body:
             self.pulp_ctx.needs_plugin(PluginRequirement("rpm", specifier=">=3.12.0"))
+        if "repo_config" in body:
+            self.pulp_ctx.needs_plugin(
+                PluginRequirement(
+                    "rpm",
+                    specifier=">=3.24.0",
+                    feature=_("A JSON document describing config.repo file."),
+                )
+            )
+        elif self.pulp_ctx.has_plugin(PluginRequirement("rpm", specifier="<3.24.0")):
+            body.pop("repo_config", None)
```

Comment:
This part is dead code. It's only reached, when `repo_config` wasn't in body in the first place.